web/js/map/ui.js

```diff
@@ -565,7 +565,8 @@ export function mapui(models, config) {
         id: def.id
       }
     });
-    if (!layer && layers[0].get('group')) {
+
+    if (!layer && layers.length && layers[0].get('group')) {
```

Comment:
Might want a `layers.length > 0` as well to ensure layers[0] exists

Reply:
`layers.length` and `layers.length > 0` are the same. The former being slightly faster.
ref: https://stackoverflow.com/questions/32911424/array-length-vs-array-length-0

Reply:
Whoops. Did not know. 👍

Reply:
All good 😉 let me try rebasing on the new tour story and see if I can reproduce